src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -1,6 +1,7 @@
 (ns status-im2.contexts.chat.messages.content.text.style
-  (:require
-    [quo2.foundations.colors :as colors]))
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.foundations.typography :as typography]
+            [status-im.ui.components.react :as react]))
```

Comment:
status-im shouldn't be used in status-im2

Reply:
Oh, Missed the import .. Okay